packages/react/src/components/Stepper/index.test.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Stepper, { Step } from './';
+import axe from '../../axe';
+
+test('should render ol element', () => {
+  render(
+    <Stepper>
+      <Step status="complete">Foo</Step>
+      <Step status="current">Bar</Step>
+    </Stepper>
+  );
+  expect(screen.getByRole('list')).toBeInTheDocument();
+});
+
+test('should have two li elements', () => {
+  render(
+    <Stepper>
+      <Step status="complete">Foo</Step>
+      <Step status="current">Bar</Step>
+    </Stepper>
+  );
+  expect(screen.getAllByRole('listitem')).toHaveLength(2);
+});
+
+test('should have specified class and data attribute on ol element', () => {
+  render(
+    <Stepper className="Foo" data-foo="7">
+      <Step status="complete">Foo</Step>
+    </Stepper>
+  );
+  expect(screen.getByRole('list')).toHaveClass('Foo');
+  expect(screen.getByRole('list')).toHaveClass('Stepper');
+  expect(screen.getByRole('list')).toHaveAttribute('data-foo', '7');
+});
+
+test('should render li element with span and text', () => {
+  render(
+    <Step status="current">
+      <span>Foo</span>
+    </Step>
+  );
+  expect(screen.getByRole('listitem')).toBeInTheDocument();
+  expect(screen.getByText('Foo')).toBeInTheDocument();
+  expect(screen.getByRole('listitem')).toContainElement(
+    screen.getByText('Foo')
+  );
+  expect(screen.getByText('Foo')).toHaveTextContent('Foo');
```

Comment:
I think we can omit this one, it seems a little strange to get the element by its text then validate that it has that text. The `toBeInTheDocument` above should cover us.

```suggestion
```